### PR TITLE
iOS Banner Crash fix

### LINKED ISSUE #33: Crashing on iOS: Trying to embed a platform view but the PaintContext does not support embedding
**ERROR**

```
Unsupported value: Error Domain=com.google.admob Code=8 "Request Error: Cannot find an ad network adapter for mediation." UserInfo={NSLocalizedDescription=Request Error: Cannot find an ad network adapter for mediation.} of type GADRequestError
[VERBOSE-2:platform_view_layer.cc(28)] Trying to embed a platform view but the PaintContext does not support embedding
*** First throw call stack:
(
	0   CoreFoundation                      0x00000001100126fb __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x000000010f5b6ac5 objc_exception_throw + 48
	2   CoreFoundation                      0x0000000110012482 +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x000000010b05b927 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 194
	4   Flutter                             0x0000000109233f6e -[FlutterStandardWriter writeValue:] + 1958
	5   Flutter                             0x00000001092329c8 -[FlutterStandardMethodCodec encodeMethodCall:] + 136
	6   Flutter                             0x000000010922396d -[FlutterMethodChannel invokeMethod:arguments:] + 58
	7   Runner                              0x00000001050dfab1 $s13admob_flutter11AdmobBannerC6adView_27<…>
[VERBOSE-2:platform_view_layer.cc(19)] Trying to embed a platform view but the PrerollContext does not support embedding
Lost connection to device.
Exited (sigterm)
```

### PR DESCRIPTION
**Pull request by kmcgill88**
I believe this will resolve #33 and #32

cc: @nemoryoliver @mmahkamov

### CODE REVIEW

File: ios/Classes/AdmobBanner.swift [hunk 1]
```diff
@@ -111,7 +111,10 @@ extension AdmobBanner : GADBannerViewDelegate {
     }
     
     func adView(_ bannerView: GADBannerView, didFailToReceiveAdWithError error: GADRequestError) {
-         channel.invokeMethod("failedToLoad", arguments: error)
+        channel.invokeMethod("failedToLoad", arguments: [
+            "errorCode": error.code,
+            "error": error.localizedDescription
+        ])
```
Comment (kmcgill88) [inline ios/Classes/AdmobBanner.swift]: This is what caused the banner crash. Dart expects a `Map`, this is more like it.

File: example/lib/main.dart [hunk 2]
```diff
@@ -56,6 +56,9 @@ class _MyAppState extends State<MyApp> {
       case AdmobAdEvent.closed:
         showSnackBar('Admob $adType Ad closed!');
         break;
+      case AdmobAdEvent.failedToLoad:
+        showSnackBar('Admob $adType failed to load. :(');
+        break;
```
Comment (kmcgill88) [inline example/lib/main.dart]: I wanted to see a visual of any errors so I added this too. 😄

**Comment by YoussefKababe** [comment]
@kmcgill88 Awesome? Let's bump the version and release?

**Comment by kmcgill88** [comment]
@YoussefKababe yup! `0.3.1` I think since this is just a bug fix.

**Comment by YoussefKababe** [comment]
@kmcgill88 Released to pub under `0.3.1`. 🎉 Thanks!

**Comment by kmcgill88** [comment]
you da man! 🥇

**Pull request merged**